paper/user_study.tex

```diff
@@ -0,0 +1,66 @@
+Our user study is mainly based on the qualitative research.
+We conduct several interviews with the professors and students as potential customers.
+We collect feedbacks from the customers, improve user experiences and fix bugs according to the feedback.
+We also use the survey to get interviewees' opinions during the user study.
+
+\subsection{Methodology}\label{subsec:methodology}
+
+\paragraph{Potential Users}
+We look for potential users to conduct the interviews.
+There are two basic standards we follow to find a potential user:
+\begin{itemize}
+    \item Professors and students who are interested in our application.
+    \item People who frequently use \cfgfiles and schema languages
+\end{itemize}
+Because each interviewee has different prior knowledge of JSON and JSON schema, we improve our interview questions and the example data also after the interview.
```

Comment:
Do we? I think not really?

Reply:
We simplified our questions once, but It's not important to write this here I think.
So I'll delete it.